sandbox/registration-processor-mz.properties

```diff
@@ -550,6 +550,13 @@ mosip.regproc.quality.checker.server.port=9072
 mosip.regproc.quality.checker.eventbus.port=5727
 mosip.regproc.quality.checker.server.servlet.path=/registrationprocessor/v1/qualitychecker
```

Comment:
This URL also we should change to qualityclassifier

Reply:
Need to understand in code where this will impact

Reply:
Again same constructed using prefix in some common code, the properties in bootstrap.properties is not considered since stages are just libraries and not apps